### PR TITLE
Autodoc the shuttle

### PR DESCRIPTION
**Pull request by spookydonut**
I dont have easily linkable examples like oranges

Also removes the shuttle loading template

### CODE REVIEW

File: code/modules/shuttle/docking.dm [hunk 1]
```diff
@@ -1,207 +1,239 @@
-/// This is the main proc. It instantly moves our mobile port to stationary port `new_dock`.
```
Comment (SpaceManiac) [inline code/modules/shuttle/docking.dm]: please undestroy the line endings

File: code/modules/shuttle/shuttle.dm [hunk 2]
```diff
@@ -1,861 +1,1123 @@
-//use this define to highlight docking port bounding boxes (ONLY FOR DEBUG USE)
```
Comment (SpaceManiac) [inline code/modules/shuttle/shuttle.dm]: ditto line endings

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

File: _maps/map_files/generic/CentCom.dmm [hunk 3]
```diff
@@ -18036,10 +18036,6 @@
 	},
 /turf/open/floor/plasteel,
 /area/centcom/supplypod/loading/four)
-"VX" = (
-/obj/effect/landmark/shuttle_import,
```
[the remark this answers is not in the document]
Comment (spookydonut) [reply _maps/map_files/generic/CentCom.dmm]: I did delete it.

File: code/modules/shuttle/docking.dm (hunk 1 above)
Comment (spookydonut) [reply code/modules/shuttle/docking.dm]: Oh did my client change them to /n/r ?

File: _maps/map_files/generic/CentCom.dmm (hunk 3 above)
[the remark this answers is not in the document]
Comment (SpaceManiac) [reply _maps/map_files/generic/CentCom.dmm]: Must've missed it

**Comment by actioninja** [comment]
@spookydonut please finish this

**Comment by spookydonut** [comment]
> 
> 
> @spookydonut please finish this

Sorry been uber busy, will try to finish it before the weekend

**Pull request closed without merging**